GitUI/CommandsDialogs/FormPull.cs

```diff
@@ -1163,6 +1164,15 @@ private void PruneTags_CheckedChanged(object sender, EventArgs e)
             AllTags.Checked = AllTags.Checked || PruneTags.Checked;
         }
 
+        private void DisableMergeOptionsWhenCurrentBranchIsLocal()
+        {
+            var isLocalBranch = Module.GetRemoteBranch(localBranch.Text) == string.Empty;
```

Comment:
Any reason not to write as follows?
```suggestion
            var isLocalBranch = string.IsNullOrWhiteSpace(Module.GetRemoteBranch(localBranch.Text));
```

Reply:
I was thinking that method call inside another method is not a good idea, also the introduction of IsNull part. Can GetRemoteBranch return null and how we should handle it? In my opinion, simple equality comparison with empty is simplier.

Reply:
> call inside another method is not a good idea

There are no issues with that. If you think it may be confusing (which I don't) you could introduce a temp variable, e.g.
```cs
string remoteBranch = Module.GetRemoteBranch(localBranch.Text);
bool isLocalBranch = string.IsNullOrWhiteSpace(remoteBranch);
```

> Can GetRemoteBranch return null and how we should handle it?

It doesn't. But even if it did - what is the concern here?

> In my opinion, simple equality comparison with empty is simplier.

Comparing strings like this is not in a spirit of C#.

Reply:
Sure, I will change it.